src/components/GameItem.js

```diff
@@ -4,24 +4,44 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  constructor(props) {
+    super(props);
+    this.state = {
+      
+    }
+  }
+   
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
+  onIconClick = () => {
+    if (this.props.type === "litter") {
+      this.setState({
+      itemClicked: "spotted-litter",
+      point: this.props.clicked(1)
+      });
 
+    } else {
+      this.setState({
+        itemClicked: "spotted-nature",
+        point: this.props.clicked(0)
+      })
+    }
+  }
   render() {
     const itemStyle = {
       bottom: `${this.props.height}px`, // use props.height to offset from the bottom of screen
       zIndex: this.props.layer, // use props.layer to set z-index, so we display ontop of background
     };
 
-    // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
-
+      let icon = ItemIcons[this.props.type]
+  
     return (
-      <div className="game-item" style={itemStyle}>
-        <img src={icon} alt="Item" className="icon-item"></img>
-      </div>
+
+        <div onClick={this.onIconClick} className={`game-item ${this.state.itemClicked}`} style={itemStyle}>
```

Comment:
I wonder if there's a more descriptive name for these styles that describe `spotted-nature` etc besides `itemClicked`?